finetune/base_models/__init__.py

```diff
@@ -24,8 +24,8 @@ def get_encoder(cls, **kwargs):
         return cls.encoder(**kwargs)
 
     @classmethod
-    def get_featurizer(cls, X, encoder, config, train=False, reuse=None):
```

Comment:
This needs to be updated for the other featurizers

Reply:
Just GPT is fine for now

Reply:
Oh just the **kwargs -- got it.